tardis/plasma/properties/atomic.py

```diff
@@ -58,18 +59,19 @@ class Lines(BaseAtomicDataProperty):
     outputs = ('lines', 'nu', 'f_lu', 'wavelength_cm')
 
     def _filter_atomic_property(self, lines, selected_atoms):
-        return lines[lines.atomic_number.isin(selected_atoms)]
+        # return lines[lines.atomic_number.isin(selected_atoms)]
+        return lines
 
     def _set_index(self, lines):
 # Filtering process re-arranges the index. This re-orders it.
 # It seems to be important that the lines stay indexed in the correct order
 # so that the tau_sobolevs values are in the right order for the montecarlo
 # code, or it returns the wrong answer.
-        try:
-            reindexed = lines.reindex(lines.index)
-        except:
-            reindexed = lines.reindex(lines.index)
-        lines = reindexed.dropna(subset=['atomic_number'])
+        # try:
+        #     reindexed = lines.reindex(lines.index)
+        # except:
+        #     reindexed = lines.reindex(lines.index)
+        # lines = reindexed.dropna(subset=['atomic_number'])
```

Comment:
@unoebauer I'm fine with removing that part since you are right, it is wrong and obsolete. However i'd leave the line filtering in place and maybe add a commented part to `_set_index()` which mirrors the current set_index that is done in atomic.py